scripts/rf_bios_settings.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/python
+# Copyright Notice:
+# Copyright 2020 DMTF. All rights reserved.
+# License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/Redfish-Tacklebox/blob/master/LICENSE.md
+
+"""
+Redfish BIOS Settings
+
+File : rf_bios_settings
+
+Brief : This script uses the redfish_utilities module to manager BIOS settings of a system
+"""
+
+import argparse
+import redfish
+import redfish_utilities
+
+# Get the input arguments
+argget = argparse.ArgumentParser( description = "A tool to manager BIOS settings for a system" )
+argget.add_argument( "--user", "-u", type = str, required = True, help = "The user name for authentication" )
+argget.add_argument( "--password", "-p",  type = str, required = True, help = "The password for authentication" )
+argget.add_argument( "--rhost", "-r", type = str, required = True, help = "The address of the Redfish service (with scheme)" )
+argget.add_argument( "--system", "-s", type = str, help = "The ID of the system to manage" )
+argget.add_argument( "--attribute", "-a", type = str, nargs = 2, metavar = ( "name", "value" ),  help = "Sets a BIOS attribute to a new value" )
```

Comment:
Aren't there some related BIOS attributes that need to be set together in the same PATCH? I recall doing that in our Ansible module. If so, I think it can be pretty easily supported by using an `action` of `append` in the `add_argument` line (see https://docs.python.org/3/library/argparse.html#action).

That would allow calls like this:

`rf_bios_settings.py ... -a BootTypeOrder0 USBDrive -a BootTypeOrder1 HardDiskDrive ...`

Reply:
Wasn't aware of that restriction; will need to add it. Was only thinking of doing one attribute at a time for the user tool (but the library lets you set multiple if you want).

Reply:
Fixed